conda-forge.yml

```diff
@@ -2,4 +2,6 @@ channel_priority: strict
 azure:
   settings_win:
     pool:
-      vmImage: windows-2019
\ No newline at end of file
+      vmImage: windows-2019
+os_version:
```

Comment:
Sorry, I was unclear. Do not edit anything outside the recipes directory.

You need to add a new `conda-forge.yml` inside `recipes/cWB` with the os_version stuff. This new `conda-forge.yml` will be copied to the correct place when your recipe is moved to its own feedstock.